src/app/shared/data-elements-list/data-elements-list.component.ts

```diff
@@ -295,15 +295,7 @@ export class DataElementsListComponent implements AfterViewInit {
   }
 
   bulkDelete() {
-    const dataElementIdLst = [];
-    this.dataElementRecords.forEach((record) => {
-      if (record.checked) {
-        dataElementIdLst.push({
-          id: record.id,
-          domainType: record.domainType
-        });
-      }
-    });
+    const dataElementIdLst = this.dataElementRecords.filter(record => record.checked === true);
```

Comment:
Again, I like how this is reduced, could also remove the equality check as well if `checked` is a boolean.